irohad/main/application.cpp

```diff
@@ -194,6 +188,46 @@ void Irohad::dropStorage() {
   storage->reset();
 }
 
+/**
+ * Initializing setting query
+ */
+Irohad::RunResult Irohad::initSettings() {
+  auto settingsQuery = storage->createSettingQuery();
+  if (not settingsQuery) {
+    return expected::makeError("Unable to create Settings");
+  } else {
```

Comment:
I suggest to remove else branch to reduce the nested indents.

Reply:
Fixed